src/testing/slim_check/testable.lean

```diff
@@ -0,0 +1,490 @@
+/-
+Copyright (c) 2020 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Simon Hudon
+-/
+
+import testing.slim_check.sampleable
+
+/-!
+# Testable Class
```

Comment:
This is a nice explanation, thanks!

Reply:
I added more explanation. Please let me know if it's clear.

Reply:
```suggestion
# `testable` Class
```